### PR TITLE
Mongoid HEAD Compatibility

### PR DESCRIPTION
**Pull request by phyllisstein**
Updates the dependency on `rails-observers` and tests the Mongoid functionality against Mongoid HEAD (v6). Bumping these dependencies will facilitate work on digitalplaywright/mongoid-slug to add compatibility for Mongoid 6.

### CODE REVIEW

File: mongoid-observers.gemspec [hunk 1]
```diff
@@ -18,7 +18,6 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rails-observers", "~> 0.1.2"
```
Comment (chamnap) [inline mongoid-observers.gemspec:21]: This gem entirely depends on `rails-observers`. Remove this line, the gem user needs to add this gem manually in their Gemfile.

**Comment by chamnap** [comment]
@phyllisstein, can you make the build pass? do you know when mongoid 6 is going to release?

File: mongoid-observers.gemspec (hunk 1 above)
Comment (phyllisstein) [inline mongoid-observers.gemspec:21]: Ah, that reflects a misunderstanding on my part---I thought adding it to the gem's `Gemfile` would be sufficient. Looks like its maintainers will need to be prodded for a fresh release.

**Comment by phyllisstein** [comment]
Not sure what the release timeline is, but I'll definitely keep working on getting the tests green again when I can! With any luck I'll have some time this week to look at it.

**Comment by Fudoshiki** [comment]
According to Mongoid JIRA, release in May 2017.

https://github.com/digitalplaywright/mongoid-slug/issues/216

**Pull request merged**